src/main/webapp/script.js

```diff
@@ -67,3 +67,10 @@ function triggerLoading() {
     let loadingState = document.getElementById(loadingId).style.display;
     document.getElementById(loadingId).style.display = loadingState === "none" ? "inline-block" : "none";
 }
+
+function checkEnterKeytoSearch() {
```

Comment:
super nit/optional: could move this function to above `displayCodes()` so that functions only call other functions below it in the file.